manticore/platforms/evm.py

```diff
@@ -67,6 +67,18 @@ def globalfakesha3(data):
     return None
 
 
+consts.add(
+    "gas",
+    default="static",
+    description=(
+        "Control how to keep the gas count"
+        "insane: Keep static and dynamic gas including transaction fee VERY EXPENSIVE (INSANE)."
```

Comment:
Nit, but can we make this "exhaustive"? Or, if you want to preserve the suggestion that it's over-the-top, perhaps "ludicrous"?